test/integration/daemonset/util.go

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
What's this change for?

Reply:
Okay. Let's just do one thing in this PR.

Reply:
Noted. I will revert the codes when squashing. Next commit will focus on `e2e`.